release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2780,3 +2780,30 @@ link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-31"]
+=== RHBA-2022:1605 - {product-title} 4.9.31 bug fix update
+
+Issued: 2022-05-02
```

Comment:
I noticed that both of the advisories have the issue date 2022-05-03. You could potentially update this Release Notes entry to the 3rd May too.